### ISSUE TITLE
Rides deleted from Strava are not deleted in database upon sync

### ISSUE THREAD

**Issue by xivind** [xivind/velo-supervisor-2000#132]
This is a bit of a corner case, but should still be handled

**Comment by thomiz** [comment]
Correct, @xivind  two different issues causing calculation errors.

**Comment by thomiz** [comment]
A simple solution would be to make it possible for manual update of all rides in the database to force VS2000 to check all rides in the database for updates. This solution is more of a quick and dirty fix because the user have to detect errors and ask for an update by pressing a button in the configuration screen or something.

Some webhook functionality is probably a lot better and more solid way to solve the use cases in question.

**Comment by xivind** [comment]
I agree @thomiz , this could be useful and I thinks its a good approach! I will follow up on that idea and get back to you. Created an issue for it #183 A bit down the road we will get webhooks, but for now this will at least enable users to correct the problem, if it arises.
